lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-max-width: 300px;
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 72px;
+$nx-sidebar-logo-height: 34px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-hover-color: $nx-indigo-600;
+
+$nx-sidebar-focus-border: 1px solid $nx-focus-border-color;
+
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar-operable {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  color: $nx-sidebar-text-color;
+  font-size: $nx-font-size;
+  max-width: $nx-sidebar-max-width;
+  padding: $nx-spacing-l 0;
+  width: $nx-sidebar-width__open;
+
+  .nx-text-link {
+    color: $nx-sidebar-text-color;
+  }
+}
+
+.nx-page-sidebar-operable__header {
+  @include container-horizontal;
+
+  display: flex;
+  padding: $nx-spacing-sm $nx-spacing-l $nx-spacing-xl $nx-spacing-l;
```

Comment:
Our padding/margin guidelines are to generally avoid padding on borderless, backgroundless elements and to prefer margin instead.  Of course, this is the first element within its parent which means any top margin you give it will be removed by `container-vertical`.  So instead of top margin (or padding) on this element I think you should increase the top padding on the sidebar itself to the desired 32px.  Then the bottom padding on this element would be converted to margin, and the side padding would either be converted to margin or just removed depending on what happens with [my side padding comment above](https://github.com/sonatype/sonatype-react-shared-components/pull/325/files#r594355921)

Reply:
> increase the top padding on the sidebar itself to the desired 32px. Then the bottom padding on this element would be converted to margin, and the side padding would either be converted to margin or just removed depending on what happens with [my side padding comment above](https://github.com/sonatype/sonatype-react-shared-components/pull/325/files#r594355921)

Done in b5fa5a7e2571d2544f9168368a76feeb026243c7